CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 - handler-sensu-deregister.rb: Fix undefined variable in case of API error.
+- check-aggregates.rb: Changed the default behaviour to alert with the severity of the aggregated checks.
```

Comment:
Breaking change, please break out into a header of `### Breaking Change` under `## [Unreleased]`.